app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -251,9 +236,16 @@ class EditBookmarkFragment : Fragment(R.layout.fragment_edit_bookmark) {
                 }
             }
         }
-    }
+        progress_bar_bookmark.visibility=View.INVISIBLE
+
+        FenixSnackbar.makeWithToolbarPadding(requireView(), FenixSnackbar.LENGTH_SHORT)
```

Comment:
I also don't think we need a snackbar for this. Sorry again about the confusion!